components/ui/toast/useToast.tsx

```
@@ -100,7 +101,7 @@ export const useShowToast = () => {
               </Toast>
             );
 
-          case TOAST.INFO:
+          case TOAST.INFO || TOAST.WARNING:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)